src/components/pages/Onboarding/OnboardingCreateAccount.tsx

```diff
@@ -67,17 +67,23 @@ class OnboardingCreateAccountView extends React.Component<
     const statusType = status ? status.status : undefined;
     switch (statusType) {
       case ApiCallStatusType.STARTED:
-        return <Text>Creating account...</Text>;
+        return <Text style={styles.paragraph}>Creating account...</Text>;
```

Comment:
Do we override the invite video when the user creates an identity video?

Reply:
nope